src/valhalla_export_edges.cc

```diff
@@ -51,13 +51,22 @@ struct edge_t {
   operator bool() const { return i.Is_Valid() && e; }
 };
 
+// Get the opposing edge - if the opposing index is invalid return a nullptr
+// for the directed edge. This should not occur but this can happen in
+// GraphValidator if it fails to find an opposing edge.
 edge_t opposing(GraphReader& reader, const GraphTile* tile, const DirectedEdge* edge) {
-  if(edge->leaves_tile())
-    tile = reader.GetGraphTile(edge->endnode());
+  const GraphTile* t = edge->leaves_tile() ? reader.GetGraphTile(edge->endnode()) : tile;
   auto id = edge->endnode();
-  auto* node = tile->node(edge->endnode());
-  id.fields.id = tile->node(id)->edge_index() + edge->opp_index();
-  return {id, tile->directededge(id)};
+  id.fields.id = t->node(id)->edge_index() + edge->opp_index();
+
+  // Check for invalid opposing index
+  if (edge->opp_index() == kMaxEdgesPerNode) {
```

Comment:
so this is valid data right? why does this happen? a node with too many outgoing edges? we cant get opposing edges for the 9th edge leaving a node or something?